### ISSUE TITLE
No way of clearing a field set against a work item

### ISSUE THREAD

**Issue by nataliia-ckb** [microsoft/azure-boards-estimate#210]
In Settings I accidentally selected the wrong Work Item Type to update, and assigned a field to it. However, there is now no way at all to set this field back to null/blank. So I cannot clear Bug work type to have no field associated with it. I can change it to any other field, but cannot clear it. 

<img width="433" alt="Screenshot 2023-01-16 at 13 43 41" src="https://user-images.githubusercontent.com/104978004/212692641-57d3562d-a266-4a52-9ff8-b7740edd467e.png">

**Comment by AminTi** [comment]
Hi! @nataliia-ckb  Thanks for reporting, can you check if you get any errors in the browser console or network log

**Comment by nataliia-ckb** [comment]
Hi @AminTi, thank you! Just checked, don't see anything unusual apart from the error message I quoted [here](https://github.com/microsoft/azure-boards-estimate/issues/208#issuecomment-1384091258), but I don't think it's related.

Sorry, forgot to say I tried this in Chrome and Safari and I assumed it was just a UI issue. Once a field is selected, there is no X/"delete"/clear button. So I can search from the list I screenshotted and change the field to anything else (and it works absolutely fine), but no way to "reset" it. I did scroll through the list of options and they were just fields, there wasn't anything that said "blank" or "clear" etc.
